src/app/components/media/MediaComponent.js

```diff
@@ -143,94 +212,175 @@ class MediaComponent extends Component {
     media.quote = media.media.quote;
     media.embed_path = media.media.embed_path;
 
+    const {
+      fragment,
+      duration,
+      playerRect,
+      playing,
+      progress,
+      start, end, gaps,
+      scrubTo,
+      seekTo,
+      showVideoAnno,
+      time,
+    } = this.state;
+
+    const { currentUser } = this.getContext();
+
+    // console.group("MediaComponent.js");
+    // console.log(this.state);
+    // console.groupEnd();
+
     return (
-      <React.Fragment>
+      <LoadScript
+        googleMapsApiKey={window.config.googleMapsApiKey}
```

Comment:
I think the `config` should not be available in `window` to start with... but anyway, the "right" way to do it is to `import config from 'config'` and then just use `config`.

Reply:
please do this one, should be an easy-win

Reply:
but where is that `config`, I don't see it in `/src/**` nor in dependencies

Reply:
It's a horrid hack:

* We load check-web's `./config.js` in a `<script>` tag
* `./config.js` writes the global variable, `window.config`
* Webpack is configured to map `import config from 'config'` to the global variable, `window.config`

(Why something so evil? For integration tests. The integration-test runner temporarily overwrites `./config.js`, injecting a test config without a Webpack recompile.)

Reply:
fixed